vars/perfRegressionParallelPipeline.groovy

```diff
@@ -125,6 +130,15 @@ def call(Map pipelineParams) {
                                     dir('scylla-cluster-tests') {
                                         checkout scm
                                         (testDuration, testRunTimeout, runnerTimeout, collectLogsTimeout, resourceCleanupTimeout) = getJobTimeouts(params, builder.region)
+                                        base_versions_list = params.base_versions.contains('.') ? params.base_versions.split('\\,') : []
+                                        // NOTE: master and enterprise will have 1 single version, but for releases,
+                                        // we will choose automatically only the last one
+                                        supportedVersions = supportedUpgradeFromVersions(
+                                            base_versions_list,
+                                            'ubuntu-focal',
```

Comment:
Why hard-coding this one ?, and not putting it on the pipelines params ?

Reply:
first of all, to reduce complexity (we don't need more complexity for now)...
secondly, we could add another parameter that defaults to `ubuntu-focal`, but our images are ubuntu based, and i don't remember a use case where we used a different one for perf tests

Reply:
I still don't lie the idea of having something like that hardcoded in the middle of the pipeline, but let's leave it for now